casepro/backend/junebug.py

```diff
@@ -463,3 +467,42 @@ def received_junebug_message(request):
         created_on=datetime.now(), has_labels=True)
 
     return JsonResponse(msg.as_json())
+
+
+@csrf_exempt
+def receive_identity_store_optout(request):
```

Comment:
I need to bring that up actually, the Identity store expects the service to use token authentication. Is that set up on casepro? How would we get that set up and how would we generate tokens?